src/app/toolkit/info/page.tsx

```diff
@@ -1,3 +1,89 @@
-export default function Page() {
-  return <h1>this is a placeholder</h1>;
-}
+"use client";
+
+import Button from "@/ui/shared/Button";
+import { Header } from "@/ui/shared/Header";
+import { useState } from "react";
+import CategoriesInfoPage from "./CategoriesInfoPage";
+
+export default function ToolkitInfoPage() {
+  const [showCategories, setShowCategories] = useState(false);
+
+  if (showCategories) {
+    return <CategoriesInfoPage />;
+  }
+
+  return (
+    <>
+      {/* Header */}
+      <Header title="Learn More" isInfoPage={true} />
+      <div className="bg-twd-background text-white min-h-screen px-10 py-6">
+      {/* Introduction Section */}
+      <div className="mt-6">
+        <h1 className="text-2xl font-bold text-white mb-5">
+          What is this Tab About?
+        </h1>
```

Comment:
```suggestion
        <h2 className="text-xl font-bold text-white mb-5">
          What is this Tab About?
        </h2>
```
H1 is in header component, text xl to match other learn more pages